.editorconfig

```diff
@@ -35,6 +35,11 @@ dotnet_diagnostic.IDE0005.severity = error
 # Yell at people when they don't include the file header template
 dotnet_diagnostic.IDE0073.severity = error
 
+# shut IDEs up about file scoped namespaces and java-style using directives 
+# while there is authoritative style rule found in here
```

Comment:
oops, that was meant to say "while there is no authoritative style rule found in here"